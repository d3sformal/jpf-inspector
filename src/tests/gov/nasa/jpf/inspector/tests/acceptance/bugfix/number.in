commence
thread 52145144525145325642351745435632463154424