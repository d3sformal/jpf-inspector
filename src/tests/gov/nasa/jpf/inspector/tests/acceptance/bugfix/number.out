INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 hit_count<=1 instruction_type=any]
	SuT (Thread=0) executes the gov/nasa/jpf/inspector/tests/acceptance/sut/Singlepath.java:0 - invokestatic gov.nasa.jpf.inspector.tests.acceptance.sut.Singlepath.main([Ljava/lang/String;)V
ERR: Could not parse a string as a number (perhaps the number does not fit in 32 bits?)
For input string: "52145144525145325642351745435632463154424"

INFO: JPF Terminating